content/en/community/hackathons/sessions/contributing-to-unikraft/content/Practical-Work_08.-Intermediary-Check.md

```diff
@@ -3,6 +3,6 @@ Move the `work/02-task-porting/src/apps/app-kdtree` application in the `$UK_APPS
 
 If needed, provide additional flags in order to suppress the compile warnings generated by this library.
 
-**Note**: You can leave the application's `main() function empty, the resulted unikernel will just print the `Unikraft` banner.
+**Note**: You can leave the application's `main() function empty for now, the resulted unikernel will just print the `Unikraft` banner.
```

Comment:
```suggestion
**Note**: You can leave the application's `main()` function empty for now, the resulted unikernel will just print the `Unikraft` banner.
```

This had a very weird usage of backticks, I don't really know if it's intentional.

Reply:
Definitely not intentional :)
No idea how I missed it before, it messed up a lot of text, thank you.